sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InMemoryFileIndex.scala

```diff
@@ -67,7 +69,18 @@ class InMemoryFileIndex(
 
   refresh0()
 
+//  override def metadataOpsTimeNs: Option[Long] = {
```

Comment:
can we remove it?

Reply:
sorry for such mistake, already removed.